### ISSUE TITLE
Missing google_auth module

### ISSUE THREAD

**Issue by ghost** [odoo/docker#442]
Social Marketing app throws an error when trying to install it in the vanilla docker image.
The google_auth module is requuired in 16.0. This module is a requirement of the Social Marketing app.
The module should be added to the Dockerfile as an apt installed module: **python3-google-auth \\**

**Comment by lathama** [comment]
@seppedl Can you confirm that attaching to the container and installing the package fixes the issue? If so we can do a PR to add it to the installed packages.

**Comment by larmar** [comment]
Yes it works but i believe its an enterprise module social_push_notifications, but it solves the dependency problem
